topics/basics/execution/run_configurations.md

```diff
@@ -0,0 +1,163 @@
+<!-- Copyright 2000-2023 JetBrains s.r.o. and contributors. Use of this source code is governed by the Apache 2.0 license. -->
+
+# Run Configurations
+
+<link-summary>Implementing run configurations management support.</link-summary>
+
+<tldr>
+
+**Product Help:** [Run/Debug Configuration](https://www.jetbrains.com/idea/help/run-debug-configuration.html)
+
+</tldr>
+
+Run Configuration is a specific type of run profile.
```

Comment:
I wanted to emphasize it is a RunProfile (which is a thing that can be executed by the execution API). What about:
> A run configuration is a specific type of run profile and a set of predefined settings that specify how a particular project or application should be executed or debugged.

?

Reply:
Ah, I did not realize that `RunProfile` is a thing. I'm not sure everyone else will know it. Do we have a link for that? (That's why to me the sentence sounded like you describe "green grass" with grass that's green).

Reply:
Oh, I see. I agree it may be unclear. We don't have a run profile section, but I linked Execution / Configuration Classes. I think it should be enough. I'll push it with the rest of the fixes.